ansible/hosts

```diff
@@ -0,0 +1,20 @@
+[infinite-staging]
+staging.infinite.industries
+
+[infinite-staging:vars]
+ansible_port=22
+ansible_user=infinite
+
+[infinite-prod]
+23.100.45.102
+
+[infinite-prod:vars]
+ansible_port=22
+ansible_user=infinite
+
+[infinite-local]
+192.168.1.43
```

Comment:
Do these IP addresses need to be hardcoded?

Reply:
yeah I think it's best to go with the ips for. I went back and fourth on this a bit, but I had an accident with DNS pointing the wrong box once that made we want to stick to ips. We can revisit as things mature